examples/ColorPicker.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import { SelectionEntry } from "../type-declarations/simularium/SelectionInterface";
+
+const ColorPicker = ({
+    uiDisplayData,
+    particleTypeNames,
+    agentColors,
+    setColorSelectionInfo,
+    updateAgentColorArray,
```

Comment:
Just peeking in, should these props have type annotations?

Reply:
I am guessing that the type checking rules are not set up to enforce this in our typecheck/linter. It would be good to turn those rules on in a separate PR!

Reply:
The example Viewer code can be a bit fast and loose compared the to the core Viewer code. I like the idea of typing it better, or even refactoring it into component files someday when there is bandwidth.

Reply:
I didn't realize this was the viewer test-app code.  Totally agree that the bar is lower for those source files, although it doesn't need to be.